bundles/org.eclipse.orion.client.ui/web/edit/setup.js

```diff
@@ -1212,6 +1203,99 @@ objects.mixin(EditorViewer.prototype, {
 					}
 				}.bind(this));
 			}.bind(this));
+			// Listen annotation changes
```

Comment:
@squarti PTAL this block of changes which handles annotation line changes and avoid duplicate events from split editor.